docs/developers/data-model.md

```diff
@@ -0,0 +1,79 @@
+## Table of Contents
+
+  - [Flow of data in OpenTrials](#flow-of-data-in-opentrials)
+  - [API database](#api-database)
+  - [Warehouse database](#warehouse-database)
+
+## Flow of data in OpenTrials
+
+1. Data about trials, publications etc. is gathered from different registries in 
```

Comment:
We might've hit a `en_US`/`en_GB` thing, is it "registry" or "register"? Need a native speaker to clear this up :)